typespec-tests/src/main/java/com/azure/lro/core/CoreAsyncClient.java

```diff
@@ -111,15 +111,7 @@ public PollerFlux<BinaryData, Void> beginDelete(String name, RequestOptions requ
      * <p><strong>Response Body Schema</strong>
      *
      * <pre>{@code
-     * {
-     *     id: String (Required)
-     *     status: String(InProgress/Succeeded/Failed/Canceled) (Required)
-     *     error: ResponseError (Optional)
-     *     result (Optional): {
-     *         name: String (Required)
-     *         resourceUri: String (Required)
-     *     }
-     * }
+     * PollResult
```

Comment:
Here problem about handling external model. Tracking issue for improvement https://github.com/Azure/autorest.java/issues/2097

Reply:
fixed https://github.com/Azure/autorest.java/pull/2098/commits/9f37bbe146cc16b89542aa6eb2a4789aeaeb7b9f